lib/formatters/srt_formatter.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Subtitles.SrtFormatter do
+  def format(subtitles) do
+    (subtitles ++ [""])
+    |> Enum.with_index(1)
+    |> Enum.map(&build(&1))
+    |> Enum.join("\n\n")
+  end
+
+  defp build({"", _}), do: ""
+  defp build({sub, idx}) do
+    ["#{idx}\n#{format_time(sub.from)} --> #{format_time(sub.to)}"]
```

Comment:
Here i would build my own index. In that way you are not dependent on the indices of the source to be correct.

This can be achieved with a map reduce.

Reply:
I am actually making my own idices in `format()` using `Enum.with_index`

Reply:
Thats nice. But i would argue that that is intrinsic to the the subtitle format rather than the subtitle cue. If you change the timing you have to change the index and then the subtitles has to be resorted.

I would move that logic to the formatter to display the order of the final subtitles.